### PR TITLE
Release 0.9.9

### PR DESCRIPTION
**Pull request by longshuicy**

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -137,7 +136,8 @@ class App extends Component {
 			viewerMenuOpen: false,
 			helpMenuOpen: false,
 			anchorEl: null,
-			errorMessage:"",
+			message:"",
```
Comment (nunezco2) [inline src/components/App.jsx:139]: Should there be a space between the colon and the double quotes?

File: src/components/DFR3Viewer.jsx [hunk 2]
```diff
@@ -393,6 +413,8 @@ class DFR3Viewer extends React.Component {
 	closeErrorMessage() {
 		this.props.resetError();
 		this.setState({
+			message:"",
```
Comment (nunezco2) [inline src/components/DFR3Viewer.jsx:416]: Same here.

**Review by nunezco2 — APPROVED** [review]
If space issues are not significant, this looks fine.

File: src/components/DFR3Viewer.jsx (hunk 2 above)
Comment (longshuicy) [reply src/components/DFR3Viewer.jsx:416]: Let's fix it in a lint specific PR?

**Pull request merged**